setup.py

```diff
@@ -10,7 +10,7 @@
     packages=find_packages(),
     zip_safe=False,
     install_requires=[
-        'django>=1.7a,<1.9',
+        'django>=1.10,<1.11',
```

Comment:
good point, and I'm fine with `>=1.10` unless it's very easy to expand the test matrix to run on more versions of Django